src/coverlet.collector/DataCollection/CoverletSettingsParser.cs

```diff
@@ -75,19 +76,20 @@ private string ParseTestModule(IEnumerable<string> testModules)
         }
 
         /// <summary>
-        /// Parse report format
+        /// Parse report formats
         /// </summary>
         /// <param name="configurationElement">Configuration element</param>
-        /// <returns>Report format</returns>
-        private string ParseReportFormat(XmlElement configurationElement)
+        /// <returns>Report formats</returns>
+        private string[] ParseReportFormats(XmlElement configurationElement)
         {
-            string format = string.Empty;
+            string[] formats = Array.Empty<string>();
             if (configurationElement != null)
             {
                 XmlElement reportFormatElement = configurationElement[CoverletConstants.ReportFormatElementName];
-                format = reportFormatElement?.InnerText?.Split(',').FirstOrDefault();
+                formats = reportFormatElement?.InnerText?.Split(',')
+                    .Select(x => string.IsNullOrEmpty(x) ? CoverletConstants.DefaultReportFormat : x).ToArray();
             }
-            return string.IsNullOrEmpty(format) ? CoverletConstants.DefaultReportFormat : format;
+            return formats == null || formats.Length == 0 ? new []{CoverletConstants.DefaultReportFormat}: formats;
```

Comment:
nit use `is` instead of `==` not fundamental btw.

Reply:
Done